library/Fission/Web.hs

```diff
@@ -94,3 +95,4 @@ server host' = Web.Swagger.server host'
           :<|> const Heroku.server
           :<|> const Auth.verify
           :<|> pure Ping.pong
+          :<|> Register.register
```

Comment:
🤔 Maybe `Register.create`, `User.register`, `User.create`, or `Register.newUser`?

I feel like `User.create` is the clearest, but would love to hear your opinions 💭

Reply:
In fact, if we make this change, then we should then also do `GET /user/verify` or similar for the current `/auth/verify` route 🤔